### ISSUE TITLE
Aliases are not generated when sklearn classes are not fitted without pipeline

### ISSUE THREAD

**Issue by cw-deenbandhu** [jpmml/jpmml-sklearn#32]
Could you please tell the difference between `objRegressor.fit(FeatureSet_df[FeatureSet_df.columns.difference(["Label"])], FeatureSet_df["Output"])`
and 
`pipeline.fit(FeatureSet_df[FeatureSet_df.columns.difference(["Label"])], FeatureSet_df["Output"])`. In the latter one alias are generated as mentioned in the dataframe in the following code. Where as in the former one alias are not generated, why is it so?

```
FeatureSet_df = pandas.read_csv("PredictiveData.csv")
objRegressor = MLPRegressor(solver='sgd', alpha=1e-6,
               hidden_layer_sizes=(10), random_state=1, max_iter=150)

objRegressor.fit(FeatureSet_df[FeatureSet_df.columns.difference(["Label"])], FeatureSet_df["Output"])

pipeline = PMMLPipeline([
  ('objRegressor', objRegressor)
])

sklearn2pmml(pipeline, "Testing.pmml")
```

**Comment by vruusmann** [comment]
The capture of feature names is `sklearn2pmml` specific functionality. Class `sklearn2pmml.PMMLPipeline` implements custom `fit` method, which checks if the `X` and `y` arguments are column name-aware Pandas data types, and if so, maps their values to `PMMLPipeline.active_fields` and `PMMLPipeline.target_field` attributes:
https://github.com/jpmml/sklearn2pmml/blob/master/sklearn2pmml/__init__.py#L32

If you invoke the `fit` method of some Scikit-Learn estimator type directly, then this name capture logic is bypassed.
